README.md

```diff
@@ -32,14 +32,16 @@ deploy:
   extend_dirs: [extend directory]
   ignore_hidden: false # default is true
   ignore_pattern: regexp  # whatever file that matches the regexp will be ignored when deploying
-
 # or this:
 deploy:
   type: git
   message: [message]
   repo:
     github: <repository url>,[branch]
-    coding: <repository url>,[branch]
+    coding:
+      url: <repository url>
+      branch: [branch]
+      token: [$ENV_TOKEN] # read your access token from environment varable with a value starting with `$`, or hard-coded plain text
```

Comment:
Coding doesn't have Personal Access Token, so this example should add below `github`.

Reply:
Also, an instruction of how to generate a Personal Access Token should be mentioned as well.